hibernate-ehcache/src/main/java/org/hibernate/cache/ehcache/SingletonEhCacheRegionFactory.java

```diff
@@ -81,6 +83,7 @@ public void start(SessionFactoryOptions settings, Properties properties) throws
 					url = loadResource( configurationResourceName );
 				}
 				final Configuration configuration = HibernateEhcacheUtils.loadAndCorrectConfiguration( url );
+				setCacheManagerNameIfNeeded( settings, configuration );
```

Comment:
This seems a bit needless.  It's using the singleton cache manager anyway.

Reply:
Yep. I thought it'd make it clearer for users, but the singleton one could be used for … god knows what else… 🔥 this